src/frontends/tensorflow/src/op_table.cpp

```diff
@@ -102,6 +102,7 @@ const std::map<std::string, CreatorFunction> get_supported_ops() {
         // note: BinaryOp translator declaration for each op must to be added in binary_op.cpp file
         {"Add", CreatorFunction(translate_binary_op<opset8::Add>)},
         {"AddV2", CreatorFunction(translate_binary_op<opset8::Add>)},
+        {"Atan2", CreatorFunction(translate_binary_op<opset8::Atan>)},
```

Comment:
We don't need template for binary translators since it is not suitable and you implemented individual translator
```suggestion
        {"Atan2", CreatorFunction(translate_atan2_op)},
```